### DISCUSSION TITLE
用whl包调用paddleocr，为什么没有产生output文件夹？

### DISCUSSION THREAD

**Discussion by dijkstra-mose** [PaddlePaddle/PaddleOCR#12365] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Ubunt22.04
- 版本号/Version：Paddle：  PaddleOCR：2.7.0.3 问题相关组件/Related components：
- 运行指令/Command Code：paddleocr --image_dir=001122.jpg
- 完整报错/Complete Error Message：运行正常，但没有输出ouput文件夹

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by cuicheng01** [comment]
不太正常呢，默认存图是打开的，看下控制台有没有输出预测结果呢

**Comment by ZhangYuef** [comment]
遇到同样的问题：控制台中有输出的预测结果，但output文件夹没有产生，也没有docx文件生成

加入 `--type structure --recovery true --use_pdf2docx_api true` 参数，可以输出待有排版的 docx 文件
